go.mod

```diff
@@ -4,6 +4,7 @@ go 1.13
 
 require (
 	github.com/Masterminds/semver v1.5.0
+	github.com/Masterminds/semver/v3 v3.1.0 // indirect
```

Comment:
This is surprising to see in this set of changes 🤔

Reply:
This is a demo commit to show the error in `check-diff` doesn't leak somewhere else in the CI. I will revert them after review is done.